controllers/controllers.go

```diff
@@ -84,6 +111,46 @@ func (r *instanceReconciler) ensureInstanceIsUpToDate(instanceInfo *instance.Inf
 	return nc.Configure()
 }
 
+var OverrideLabel = "windowsmachineconfig.openshift.io/force-upgrade"
```

Comment:
are there other reasons why we might force an upgrade? since this is generic (i.e. not storage-specific), I'm wondering if what we're doing here is "adding a general upgrade force label" that _right now_ only happens to apply to the CSI storage block?